deploy/helm/prometheus-overrides.yaml

```diff
@@ -193,6 +193,21 @@ prometheus:
       ## kube_node_status_allocatable
       ## kube_node_status_capacity
       ## kube_node_status_condition
+      ## kube_statefulset_metadata_generation
+      ## kube_statefulset_replicas
+      ## kube_statefulset_status_observed_generation
+      ## kube_statefulset_status_replicas
+      - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics.state
+        writeRelabelConfigs:
+          - action: keep
+            regex: kube-state-metrics;(?:kube_statefulset_status_observed_generation|kube_statefulset_status_replicas|kube_statefulset_replicas|kube_statefulset_metadata_generation|kube_daemonset_status_current_number_scheduled|kube_daemonset_status_desired_number_scheduled|kube_daemonset_status_number_misscheduled|kube_daemonset_status_number_unavailable|kube_deployment_spec_replicas|kube_deployment_status_replicas_available|kube_deployment_status_replicas_unavailable|kube_node_info|kube_node_status_allocatable|kube_node_status_capacity|kube_node_status_condition)
+            sourceLabels: [job, __name__]
+          - action: labelmap
+            regex: (pod|service)
+            replacement: sumo_metrics_${1}
```

Comment:
Nit.  sumo_metrics_ as the prefix feels unintuitive. In this case, `pod` and `service` are pointers to the pod/service that Prometheus scraped the data from.  Let's change it `sd_` to indicate it is part of the service discovery mechanism.

Reply:
Fixed

Reply:
I think we should go further and use `service_discovery_${1}` - no need to use the non-obvious abbreviation.